pkg/kubelet/apis/well_known_labels.go

```diff
@@ -18,13 +18,14 @@ package apis
 
 const (
 	LabelHostname          = "kubernetes.io/hostname"
-	LabelZoneFailureDomain = "failure-domain.beta.kubernetes.io/zone"
-	LabelZoneRegion        = "failure-domain.beta.kubernetes.io/region"
+	LabelZoneFailureDomain = "failure-domain.kubernetes.io/zone"
+	LabelZoneRegion        = "failure-domain.kubernetes.io/region"
 
-	LabelInstanceType = "beta.kubernetes.io/instance-type"
+	LabelInstanceType = "kubernetes.io/instance-type"
 
-	LabelOS   = "beta.kubernetes.io/os"
-	LabelArch = "beta.kubernetes.io/arch"
```

Comment:
it's safe to assume there are workloads in the real world that assume the presence of these labels today (with the `beta` keys). when combined with making controller selectors immutable (#50808 & #50719), I'm trying to think through the implications of changing these, and how someone would modify their existing workloads to use the non-beta labels non-disruptively. do we need to announce deprecation of beta labels, and set both beta and non-beta labels for some period? cc @kubernetes/api-reviewers

Reply:
I don't think this is safe within our definitions of API stability.  This breaks anyone using those labels without warning.

Reply:
@liggitt @smarterclayton and others, I have created a issue (https://github.com/kubernetes/kubernetes/issues/51756) for the `beta` label.